### ISSUE TITLE
[Task] Lenra Metrics

### ISSUE THREAD

**Issue by jonas-martinez** [lenra-io/lenra_cli#418]
## What should be done 

This issue was opened because we need more metrics on Lenra. I will try to list all metrics that are missing on the current state of Lenra.

### Metrics list

The metrics are sorted in order of importance. The first ones are rated 5 on a scale of priority of 5. The last ones are rated 4. I did not include the lower priority metrics as we already have enough work with these and they will do 90% of the work.

- **Uptime/Downtime** : There is currently no way to see this metric, is it easy to do ? I suppose that this should be done on Kubernetes. Either we constantly monitor all of the pods or we do it every x days or weeks. The best solution would be to constantly monitor but it depends on the feasibility and power consumption.
- **Security Vulnerabilities** : Not easy to do. We already have dependabot that is supposed to pinpoint every vulnerability on dependencies and propose an update. This is only targeting a very narrow range of vulnerabilities, we should do a complete audit of Lenra if we really want to address the full range of vulnerabilities.
- **API Response Time (Mean)** : Which API, what endpoint, what data ?
  - Websocket response time
  - Mongo queries
- **Errors and exceptions** : 
  - Add to Grafana the number of errors per week
  - Add to Grafana the number of errors per user (number of errors per week % number of active users per week)
  - Add to Grafana the number of active users 
  - Add to Grafana the number of errors per level (critical, warning, etc...)
- **Number of users in parallel** : 
  - This is the number of users in parallel for a given period.
  - This is also a load test to see how the server reacts when given a high amount of users (JMeter)
- **CPU/RAM/disk usage** : 
  - The usage for each application
  - The usage of the whole server
- **Number of applications deployed** : We need to get the number of applications deployed that are not made by any of us. We could take into account the Berlioz & Molny projects.
- **Application loading time** : The time the application takes to start and be ready to use.
